update-db.js

```diff
@@ -158,4 +200,32 @@ module.exports = function updateDB (print) {
   childProcess.execSync(del + ' caniuse-lite')
 
   print('caniuse-lite has been successfully updated\n')
+
+  var currentBrowsersList
+  if (!browsersListRetrievalError) {
+    try {
+      currentBrowsersList = getBrowsersList()
+    } catch (e) /* istanbul ignore next */ {
```

Comment:
It's hard to cause error only on the second call to `getBrowsersList()` in tests, but i think such situation is possible.

Reply:
I agree 👍